packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,87 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<DropdownWrapper>
+  <Button data-toggle="dropdown" data-target="DropdownExample" aria-controls="DropdownExample" aria-expanded="false">Open Dropdown</Button>
+  <Dropdown id="DropdownExample">Dropdown Content</Dropdown>
+</DropdownWrapper>
+```
+
+Responsive usage for tablet
+
+```twig
+<DropdownWrapper>
+  <Button data-toggle="dropdown" data-target="DropdownExample" aria-controls="DropdownExample" aria-expanded="false">Open Dropdown</Button>
+  <Dropdown id="DropdownExample" breakpoint="tablet">Dropdown Content</Dropdown>
+</DropdownWrapper>
+```
+
+Advanced example usage with positioning:
+
+```twig
+<DropdownWrapper>
+  <Button data-toggle="dropdown" data-target="DropdownExample" aria-controls="DropdownExample" aria-expanded="false">Open Dropdown</Button>
+  <Dropdown elementType="span" id="DropdownExample" class="custom-dropdown-class" placement="top-right" isFullWidth>Dropdown Content</Dropdown>
+</DropdownWrapper>
+```
+
+Without lexer:
+
+```twig
+{% embed "@spirit/dropdownWrapper.twig" %}
+  {% block content %}
+    {% embed "@spirit/dropdown.twig" with { props: {
+        breakpoint: 'tablet',
+        class: 'dropdown-custom-class',
+        elementType: 'span',
+        isFullWidth: true,
+        placement: 'top-right'
+    }} %}
+        {% block content %}
+              Dropdown content
+        {% endblock %}
+    {% endembed %}
+  {% endblock %}
+{% endembed %}
+```
+
+You can add any custom trigger like a `<Button>` but it is necessary to add `data-toggle="dropdown"`, `data-target="<id>"`
+attributes to register trigger events.
+
+## API
+
+### Dropdown
+
+| Prop name     | Type                                                     | Default       | Required | Description                               |
+| ------------- | -------------------------------------------------------- | ------------- | -------- | ----------------------------------------- |
+| `id`          | `string`                                                 | -             | yes      | Dropdown ID                               |
+| `breakpoint`  | `string`                                                 | -             | no       | Breakpoint level [mobile,tablet,desktop]  |
+| `class`       | `string`                                                 | `null`        | no       | Custom CSS class                          |
+| `elementType` | `string`                                                 | `div`         | no       | HTML tag to render                        |
+| `isFullWidth` | `boolean`                                                | `false`       | no       | Whether is component viewed in full width |
```

Comment:
I think _displayed_ is better: _I can view what is displayed_ 🙂.

```suggestion
| `isFullWidth` | `boolean`                                                | `false`       | no       | Whether is component displayed in full width |
```

Reply:
Fixed.